src/pages/404.js

```diff
@@ -2,62 +2,100 @@ import React from "react"
 import Layout from "../layout/layout"
 import SEO from "../layout/seo"
 import SVG404 from "../../source/images/404_dark.svg"
+import { Link, graphql } from "gatsby"
```

Comment:
@rachelwhitton is it accurate that the 404 behavior you wanted to get working just isn't working? If so, I'd rather not add non-working code to this file